unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/FollowBehavior.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FollowBehavior : MonoBehaviour
```

Comment:
We could make use of this and remove the `SkyboxCameraBehaviour`.

Also:
```suggestion
public class FollowBehaviour : MonoBehaviour
```

Reply:
SkyboxCameraBehaviour has additional work of catching the FOV of the camera. So that was the reason to have that separate.